my-dsl.md

```diff
@@ -2,37 +2,60 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Mathematica/Wolfram Alpha](http://www.wolfram.com/mathematica/?source=nav)
+
 
 # Domain
 _Describe the language's domain in five words._
 
+To create mathematical figures/expressions
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+When a program runs, a text parser sends statements to be evaluated to a mathematical figure/expression engine, which are then displayed on the screen using graphics. 
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+I feel it is closer to a general-purpose language because it is able to run algorithms. For example, Mathematica does have conditional statements and loops; however, they are only useful for numbers and do not work with strings. The presence of conditional statements and loops definitely brings Mathematica close to the general-purpose side of the spectrum. 
```

Comment:
I'm of similar mind here. Mathematica (and R) are obviously designed and targeted towards specific tasks, but they really do have the capabilities to be GPLs. 

I don't disagree with your categorization, however. Just because people **can** use something as a GPL doesn't mean they **should**, and I think that if you're using Mathematica (and to a lesser extent R) as a GPL then maybe you need to rethink your tool choices.